src/main/java/org/beta/tchap/identite/bot/BotSender.java

```diff
@@ -0,0 +1,17 @@
+package org.beta.tchap.identite.bot;
+
+import org.beta.tchap.identite.matrix.rest.room.RoomService;
+
+public class BotSender {
+
+    private final RoomService roomService;
+
+    public BotSender(RoomService roomService) {
+        this.roomService = roomService;
+    }
+
+    public void sendOtp(String otp, String destMatrixId) {
```

Comment:
It could very usefull to document which RuntimeException are thrown when errors happen